txscript/standard.go

```diff
@@ -153,7 +153,7 @@ func isPubKeyScript(script []byte) bool {
 // is a standard pay-to-pubkey-hash script.  It will return nil otherwise.
 func extractPubKeyHash(script []byte) []byte {
 	// A pay-to-pubkey-hash script is of the form:
-	//  OP_DUP OP_HASH160 <20-byte hash> OP_EQUALVERIFY OP_CHECKSIG
+	//  OP_DUP OP_HASH160 OP_DATA_20 <20-byte hash> OP_EQUALVERIFY OP_CHECKSIG
```

Comment:
The `OP_DATA_20` is kind of implied by the `<20-byte hash>`...

Reply:
It's not consistent, [here](https://github.com/btcsuite/btcd/blob/b1b94202082b8e22160df31349992f37fb360f75/txscript/standard.go#L99) is an example.